environment-upstream-dev.yml

```diff
@@ -0,0 +1,53 @@
+name: notebook-gallery
+channels:
+- conda-forge
+- defaults
+dependencies:
+- bokeh
+- bottleneck
+- cartopy
+- cf_units
+- cftime>=1.0.4.2
+- cmocean
+- dask-jobqueue>=0.7.0
+- dask>=2.7
+- eofs
+- h5py
+- holoviews
+- hvplot
+- intake-xarray
+- ipywidgets
+- jupyterlab
+- matplotlib
+- metpy
+- mpi4py
+- nbval
+- netcdf4
+- nodejs
+- numba
+- pandas
+- pandoc
+- pandocfilters
+- pip
+- pre-commit
+- psutil
+- pytest
+- python=3.7
```

Comment:
Since the Python version is specified via `image: ncarxdev/miniconda:3.8`, having `python=3.7` ends up overriding the python version available in the docker container. If you check the CI logs [here](https://app.circleci.com/pipelines/github/NCAR/notebook-gallery/114/workflows/0e542778-d509-4cec-8fa1-70f948212b50/jobs/129/parallel-runs/0/steps/0-104) and [here](https://app.circleci.com/pipelines/github/NCAR/notebook-gallery/114/workflows/0e542778-d509-4cec-8fa1-70f948212b50/jobs/128/parallel-runs/0/steps/0-104), you will notice that all the environments are using **Python 3.7**. I recommend removing `- python=3.7` from the `environment.yml` and `environment-upstream-dev.yml` files

Reply:
Thanks for catching this.

Reply:
I have removed "python=3.7" from the YML files, but this version is still being chosen for all builds.   I am going to try to make the install process verbose, to see if some other package forces the use of python 3.7